core/src/components/action-sheet/test/basic/index.html

```diff
@@ -114,7 +119,7 @@
               buttons: [{
                 text: 'Okay',
                 handler: async () => {
-                  await actionSheetElement.dismiss();
+                  await window.actionSheetController.dismiss();
```

Comment:
```suggestion
                  await actionSheetController.dismiss();
```

You shouldn't need window on any of these